scripts/sanity_chk/harness.py

```diff
@@ -121,6 +127,29 @@ def handle(self, line):
 
         self.process_test(line)
 
+class Pytest(Harness):
+    def handle(self, line):
+        if self.handle_script and self.BOOTING_ZHEPHYR in line:
+            with subprocess.Popen([self.handle_script, self.running_dir], stdout=subprocess.PIPE) as proc:
+                try:
+                    stdout, _ = proc.communicate(15)
```

Comment:
this is not going to be a robust way for collecting results. I would run the script using pytest, capture results in a stanrdized way (for example using `--junitxml=report.xml` and then read that file and get the results)

Reply:
fixed,   used --junitxml to save report and use  ElementTree to check the results.   thanks.